setup/install.sh

```diff
@@ -25,11 +25,10 @@ cur_dir=$(dirname $0)
 
 additional(){
 
-    sudo pip3 install --upgrade pip
-    sudo pip3 install --upgrade -r $cur_dir/requirements.txt
-    
+    sudo pip install --upgrade -r $cur_dir/requirements.txt
```

Comment:
pip != pip3
we need to be sure that the tool installs the python3 dependencies and not python2

Reply:
pushed